python/tvm/relay/op/nn/_nn.py

```diff
@@ -142,6 +142,11 @@ def alter_op_layout_sparse_dense(attrs, inputs, tinfos, out_type):
     return topi.nn.sparse_dense_alter_layout(attrs, inputs, tinfos, out_type)
 
 
+# sparse_add
+reg.register_strategy("nn.sparse_add", strategy.sparse_add_strategy)
+reg.register_pattern("nn.sparse_add", reg.OpPattern.ELEMWISE)
```

Comment:
I think this should be `reg.OpPattern.OPAQUE` instead of `reg.OpPattern.ELEMWISE`. I don't think te.extern is fusable.